build.gradle

```diff
@@ -200,10 +206,18 @@ def depTehuti = 'io.tehuti:tehuti:0.7.0'
 
 
 shadowJar {
+    zip64 true  // Required if fatjar has more than 64K files
     classifier "bnp"
     from sourceSets.main.output, sourceSets.test.output, sourceSets.main.resources
     from {subprojects*.sourceSets.main.output}
     from {subprojects*.sourceSets.test.output}
+    from project('gobblin').projectDir
+    dependsOn 'gobblin:shadowJar'
+
+    // Hadoop dependencies are expected to be provided by Azkaban.
+    // If Azkaban is not included in your deployment, you may need to remove the following excludes
+    exclude("**/org/apache/hadoop/**")
+    exclude("**/org.apache.hadoop**")
```

Comment:
I copied from voldemort-bnp in voldemort-build-and-push MP.
If I remember right, this line is necessary.